### DISCUSSION TITLE
Problems with Docker

### DISCUSSION THREAD

**Discussion by Pri5m4** [bellingcat/auto-archiver#131] [Q&A]
Hey, I installed auto-archiver via Docker and setup all the required files. 
When i run "docker run" i get  "SUCCESS  | auto_archiver.feeders.gsheet_feeder:__iter__:79 - Finished worksheet 1" but the worksheet is not being updated, and no files are downloaded. I tried tiktok_archiver and youtubedl_archiver.  Can you help me out?

**Comment by msramalho** [comment]
Hey @Pri5m4 , thanks for opening an issue!

If you got to the `SUCCESS` message that means the auto-archiver setup is okay in docker, the issue is likely with your configuration file.
Can you share a screenshot of the spreadsheet header? and what configuration values you are using for the `gsheet_feeder.columns` and `gsheet_feedr.sheet` name? If you share more of the logs it will be easier to debug your issue. 

Additionally, do you see a `SKIPPED worksheet ` message in the logs?
